generators/filesystem/testing/expected.csv

```diff
@@ -0,0 +1,46 @@
+entity,attribute,value
+foo.json,id,/Users/swanickj/Code/origins-generators/generators/filesystem/testing/test_dir/foo.json

```

Comment:
This test will fail on any else's machine since it depends on the absolute path. I think it would be better to make the paths relative as well.